src/utils/validation/getIdentifierType.ts

```diff
@@ -0,0 +1,32 @@
+const esdtParts = 2;
+const nftParts = 3;
+
+export function getIdentifierType(identifier?: string): {
+  isEsdt: boolean;
+  isNft: boolean;
+  isEgld: boolean;
+} {
+  const result = {
```

Comment:
this can probably be pulled into a constant object outside the component, like initialResult